WrightTools/data/_data.py

```diff
@@ -861,8 +861,10 @@ def create_channel(
                 require_kwargs["dtype"] = np.dtype(np.float64)
             else:
                 require_kwargs["dtype"] = dtype
-            if require_kwargs["dtype"].kind in "fcmM":
+            if require_kwargs["dtype"].kind in "fmM":
                 require_kwargs["fillvalue"] = np.nan
+            elif require_kwargs["dtype"].kind in "c":
+                require_kwargs["fillvalue"] = np.complex128(np.nan)
```

Comment:
I think you want to fill with `np.complex(np.nan, np.nan)`.  

Comment applies to filling all blank arrays.